align tabs, app specific tab coloring,  restore caja sidebar color,  improve pane separator.

mdinslage opened the pull request:
I played around with different ways to align all the tabs with the frame correctly, but the below code was what I found to correctly align all programs.  I tried various ways but there was always a program or too that was just a little off.  This code below should correctly align all programs, unless I missed checking one.  

I restored the caja side panel coloring like it was in gtk-3.18.

I changed the checked/selected tabs in pluma and mate-terminal to theme-base-color and made the bottom border transparent, this way they correctly blend into the frame window seamlessly. 

I restored the frame border to pulseaudio volume control.

And lastly I restored the pane separator to its 3.18 look, it now has the correct icon and spacing, this looks right on every program I checked, but please review the code, because I wasn't 100% sure this was the best way to restore it the way it looked in 3.18.

JosephMcc commented:
I don't think your pane separator changes are correct. There are two types in gtk. Wide and normal. If an app wants the wide style of separator it should ask for it. Also the images used for paned separators are already defined in gtk-widgets-assets.css. You probably shouldn't be redefining them here. That gets confusing.

mdinslage commented:
Ok Thank you, Like I mentioned I wasn't 100% sure if this was the best way to go about it.  I'll work on it some more, or maybe someone will get to it before me.

mdinslage commented:
fixed the border-radius mistake and just reverted the "pane separator" code for now.  I'll work on that separately. For now just sticking with the tab fixes and caja side pane.
